docs/courses/classes.md

```diff
@@ -1030,7 +1031,7 @@ def main():
   grade = random.randint(10, 100)
   # Send the grade back to Codio with the penatly factor applied
   
-  res = send_grade_v2(int(round(grade)), '### Hi here', FORMAT_V2_MD)
+  res = send_grade_v2(int(round(grade)), feedback, extra_credit=int(round(grade)), format=FORMAT_V2_HTML)
```

Comment:
there is no `feedback` variable in this code -- this example will not work

@MaximKraev -- it looks like `format` is an optional parameter -- it's being used in a different way now -- why is it like this?